Checkout/Checkout/Test/Stubs/StubCardValidator.swift

```diff
@@ -9,6 +9,14 @@ import Foundation
 @testable import Checkout
 
 final class StubCardValidator: CardValidating {
+  var eagerValidateCardNumberToReturn: Result<Card.Scheme, ValidationError.CardNumber> = .success(.visa)
+  private(set) var eagerValidateCardNumberCalledWith: String?
```

Comment:
I wouldn't generally worry about access control in stubs. It would be clearly visible at use site if it was not handled correctly.

Reply:
Also, you guys may have discussed this before, and I know its a very pedantic detail, but these stuck are [technically mocks](https://stackoverflow.com/a/5180286) 🙈.